cmd/node/main.go

```diff
@@ -339,11 +376,20 @@ func generatePDSAppConfig(
 				Matcher: "/xrpc",
 				Target:  "http://host.docker.internal:3000/xrpc",
 			},
+			/*
```

Comment:
let's add a proxy rule specifically for /.well-known/did.json so that we can restore this